ui/src/api-logic/graphql/mutations/add-users.ts

```diff
@@ -0,0 +1,39 @@
+import { gql } from '@apollo/client'
```

Comment:
let's rename this file `manage-users.ts` since it's not just for adding users

Reply:
Done [7441bbe](https://github.com/mergestat/mergestat/pull/572/commits/7441bbe80abf9aa84a1bf2c078fc6c21f869e668)